gremlin/templates/chao-deployment.yaml

```diff
@@ -25,6 +25,15 @@ spec:
         app.kubernetes.io/version: "1"
     spec:
       serviceAccountName: chao
+      {{- if .Values.affinity }}
```

Comment:
This will couple affinities/nodeSelectors/tolerations between our `daemonset/gremlin` and our `deployment/chao`, which have fundamentally different deployment requirements. 

* `daemonset/gremlin` must be colocated with all attack targets. In other words, it must run on the same machines as the applications you with to failure test.
* `deployment/chao` needs to run in as a single instances, and may run on any node.

Do we expect there to be problems that arise from coupling these requirements?

Reply:
The alternative here would be to create 3 new fields, forked from these:

```
.Values.chao.affinity
.Values.chao.nodeSelector
.Values.chao.tolerations
```

Reply:
My preference is three new fields.